server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -134,7 +134,7 @@ public static void download(Repository repository, ShardId shardId, ThreadPool t
     }
 
     public static void download(TranslogTransferManager translogTransferManager, Path location) throws IOException {
-        logger.info("Downloading translog files from remote for shard {} ", translogTransferManager.getShardId());
+        logger.info("{} Downloading translog files from remote", translogTransferManager.getShardId());
```

Comment:
Shall we pass the logger (which was created using shardId information) into this method and use it within this method?

Reply:
this method is called from 2 places, one of from the constructor of RemoteFsTranslog(which is where we can pass the logger), another is from above method `download`, which too is static and is only called from IndexShard. So we would have to pass the logger reference from IndexShard in this case